satpy/etc/readers/omps_edr.yaml

```diff
@@ -166,4 +170,74 @@ datasets:
     units: degrees
     standard_name: latitude
 
+  Longitude:
+    name: Longitude
+    resolution: 50000
+    file_type: omps_sampo
+    file_key: GEOLOCATION_DATA/Longitude
+    units: degrees
+    standard_name: longitude
+  Latitude:
+    name: Latitude
+    resolution: 50000
+    file_type: omps_sampo
+    file_key: GEOLOCATION_DATA/Latitude
+    units: degrees
+    standard_name: latitude
```

Comment:
for consistency, should this be called `Longitude_sampo` and `Latitude_sampo` ?

Reply:
I think your and @djhoese comment on the same thing.

And here I mess up things.

- the `file_key` must be exactly the name in the hdf file, so these must be like this. OK?
- the `name` what must that be? And what can it be?
- the `standard_name` is the cf name? like `latitude`?
- and last the the name of the dataset. Is that what shows up in `available_datasets` and can be loaded?

Reply:
* file_key: correct, this shouldn't change
* name: This can technically be anything but it needs to be reflected in the `coordinates: [longitude_new_name, latitude_new_name]` for all datasets that use it. I'm specifically talking about the part that says `name: Latitude` (see last bullet).
* standard_name: correct, this doesn't need to change unless it doesn't match CF standards. Speaking of which I think units need t obe "degrees_north" and "degrees_east".
* The name of the section `Latitude:` can be whatever you want and is only used for uniqueness.